README.md

```diff
@@ -78,7 +78,7 @@ The stream will emit objects with keys of `event` and `instance`.
 
 ```js
 const onData = ({event, instance} => {
-  console.log(`${instance.$modelOptions.name.singular} had a ${event} event`)
+  console.log(`${instance._modelOptions.name.singular} had a ${event} event`)
```

Comment:
hmm… did this change in a recent sequelize version?

Reply:
i tested it in v4.2.1 and other new versions (this repo using 4.2.1), proper use seems to be _modelOptions. sorry for late answer.